### DISCUSSION TITLE
[ISSUE] Wrong join date

### DISCUSSION THREAD

**Discussion by kokonut27** [lowlighter/metrics#737] [Q&A] [answered]
# Hello!

So I checked out my metrics.svg, but for some reason it says I joined *1 day* ago, which isn't true. When you check the contributions on my profile page, it says otherwise, and I have been on GitHub for at least almost a year. Is it possible to fix this? Thanks!

#### Metrics
![Screenshot 2022-01-02 11 19 08 AM](https://user-images.githubusercontent.com/76911308/147882093-b1bf4e53-ddd5-4482-8231-59a386a1c29b.png)

#### GitHub contributions
![Screenshot 2022-01-02 11 19 52 AM](https://user-images.githubusercontent.com/76911308/147882129-594630fd-8478-43b6-90c2-221042eac0c2.png)

**Comment by lowlighter** [comment, marked as answer]
Hi!
Probably an issue with how the date is computed, the bug will be tracked by issue #738 

Thanks for your report!

**Reply by kokonut27** [reply]
Heya!

Oh, okay! Thanks!

No problem :)

**Reply by spenserblack** [reply]
Sorry about that. In implementing the bug fix for leap years, it looks like I introduced some more significant bugs 😦

**Reply by kokonut27** [reply]
Hey @lowlighter, it seems like it's been fixed now! Thanks!

**Reply by lowlighter** [reply]
I think the bug targets user with less than 1 year during their anniversary month, but since you have now 1 year you're not affected anymore 😅

**Reply by kokonut27** [reply]
Oh, I see. Good luck! Let me know if you need any help too :D

**Reply by lowlighter** [reply]
This has been fixed by @JayantGoel001 in #746 🎉 !
